test/unit/resources/elasticsearch_test.rb

```diff
@@ -0,0 +1,36 @@
+# encoding: utf-8
+# copyright: 2017
+# author: Rony Xavier, rx294@nyu.edu
+# license: All rights reserved
+
+require 'helper'
+require 'inspec/resource'
+
+describe 'Inspec::Resources::Elasticsearch' do
+  describe 'elasticsearch' do
+    it 'Verify Elasticsearch node version' do
+      resource = load_resource('elasticsearch')
+      _(resource.version).must_equal ["5.4.1"]
+    end
+
+      it 'Verify elasticsearch node mlockall state' do
```

Comment:
Indentation lines 16-34 is off.. should be in line with the block above in lines 11-14.